ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -75,11 +75,11 @@
   public final double storeIndexBloomMaxFalsePositiveProbability;
 
   /**
-   * How long (in days) a key must be in deleted state before it is hard deleted.
+   * How long (in hours) a key must be in deleted state before it is hard deleted. Minimum value: 1 hour.
    */
-  @Config("store.deleted.message.retention.days")
-  @Default("7")
-  public final int storeDeletedMessageRetentionDays;
+  @Config("store.deleted.message.retention.hours")
+  @Default("24")
```

Comment:
One minor suggestion:  maybe we can use 7 * 24 as default value which lines up with we are using for main cluster, and change this for delivery cluster only.

Reply:
sure.